discover/portal/articles/07-collaborating/01-publishing-blogs.markdown

```diff
@@ -0,0 +1,444 @@
+# Publishing Blogs [](id=publishing-blogs)
+
+Since blogs are a mainstay on the web, bloggers and blog administrators expect
+feature-rich environments for publishing blogs. Bloggers expect not only a
+powerful editor that allows them to create terrific content, but also a
+pleasurable authoring experience�like sipping a hot personalized beverage while
+touching up your final draft. Blog admins demand an intuitive environment
+that facilitates configuring blog instances and managing blog entries
+efficiently. Liferay's Blogs meets these demands. 
+
+Its editor delivers features you'd expect, in an elegant in-context manner. It
+has a complete set of WYSIWYG controls that appear where and when you need them.
+You can stay in this easy-to-use mode or switch to source mode to edit your
+content's HTML code. In source mode, you can work with light text on a dark
+background or dark text on a light background. To cap things off, you can do
+fullscreen editing to minimize distractions as your create your posts. You'll
+have a satisfying experience creating awesome blog posts. 
+
+Lastly, Liferay Blogs empowers you to show off your blogs using powerful display
+portlets. You can leverage Liferay's built-in display templates or create your
+own, to present blogs the way you like them. And you can now add a beautiful
+cover image to each of your blog entries. Their like album covers for your
+posts. Let's face it--you might not be able to judge a book by its cover, but
+you're more likely to draw readers to your blog entry by decorating it with an
+enticing cover image! 
+
+By now you're probably chomping at the bit to start blogging, right?! Let's get
+started. 
+
+## Adding Blog Entries [](id=adding-blog-entries)
+
+Each site comes with a built-in blog instance. So let's explore adding an entry
+to your site's blog. The Blogs Administration portlet provides the best place to
```

Comment:
I think that we don't have the concept of Blogs Administration portlet. I mean, we do, but from a developer point of view. From a user point of view is just blogs in Site Administration. But maybe people will understand what Blogs Administration mean.